CPs/cp_nn6.adoc

```diff
@@ -1045,7 +1044,7 @@ The management of the query and response connection between the Device-Patient A
 [.text-left]
 [underline]#Proposed Text:#
 [.text-left]
-The management of the query and response connection between the Device-Patient Association Consumer and the Device-Patient Association Manager in the case of an ongoing subscription is an implementation detail, but one practical method is for the Device-Patient Association Manager to maintain an open TCP listen port to accepts connections from one or more Device- Patient Association Consumer clients and then to open an individual TCP connection with each requester that persists as long as the client is connected and the query is valid (within its time limits, if any).
+The Device-Patient Association Manager is responsible for collecting, formatting and sending the requested information back to the querying Device-Patient Association Consumer according to the filtering specified in the query.
```

Comment:
Suggestion: "..back to the querying Device-Patient..." -> "back to the Device-Patient.."

Reply:
Thanks, changes committed.